src/common/transformations/src/transformations/common_optimizations/moc_transformations.cpp

```diff
@@ -167,6 +169,9 @@ bool ngraph::pass::MOCTransformations::run_on_model(const std::shared_ptr<ngraph
     common_fusions->add_matcher<ngraph::pass::GeluFusion>();
     common_fusions->add_matcher<ngraph::pass::LeakyReluFusion>();
     common_fusions->add_matcher<ngraph::pass::RandomUniformFusion>();
+    common_fusions->add_matcher<ov::pass::GRUCellFusion>();
```

Comment:
Can we affect other plugins except CPU and GPU by this change? They may still expect it in the decomposed form.

Reply:
yes, it looks like we need to call corresponding transformations to decompose Cells/Sequences, I'm checking it right now
but I didn't find ConvertToTensorIterator calls in other plugins, so it means if the node is Sequence, it remains as Sequence, it quite strange because in CPU/GPU we have some limitations